sorts/binary_search.cpp

```diff
@@ -0,0 +1,51 @@
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+const int32_t binarySearch(const int32_t value, const std::vector<int32_t> vector, const size_t size) {
```

Comment:
Why not make the function a template?

Reply:
I wanted to try something like that but then realized that I have no idea how to implement it.

Reply:
Why passing vector by value and not reference?

Reply:
Since the result of the function is index - why would we use signed integer?

Reply:
> Since the result of the function is index - why would we use signed integer?

If you'll take a look on previous commits, this function could return -1 if provided value couldn't be found so I forgot to fix it.